apps/server/package.json

```diff
@@ -2,21 +2,26 @@
 	"name": "server",
 	"version": "1.0.0",
 	"description": "",
-	"main": "index.js",
+	"main": "dist/index.js",
```

Comment:
🤔 my first instinct is we shouldn't be pointing at a file in a folder, and have the entry point be at the root level, but then again I'm thinking we could just share the source code instead, and use ts-node.

ultimately there's two different exports or entry points here: as a standalone server, and as dependency library.
meaning we can address this in a later PR.

Reply:
just figured out we won't require this main property in the server/ so I just removed.